kurtosis-devnet/isthmus.yaml

```diff
@@ -31,7 +29,7 @@ optimism_package:
         tolerations: []
         count: 1
       - el_type: op-reth
-        el_image: "ghcr.io/paradigmxyz/op-reth:nightly@sha256:cc2fce901755783b42f5c76b8adc81375c7a80b8ca080b76b6e35cfb4d3e41a7"
+        el_image: "ghcr.io/paradigmxyz/op-reth:nightly"
```

Comment:
Could we pin on a specific version here? Otherwise this may break without warning.

Reply:
See https://github.com/ethereum-optimism/optimism/pull/15122/commits/1924535af8db3b39fa46c105ce2cad6fc8f6a7d0

Fetched my local op-reth's image(the test-working version) hash via 
> `docker inspect ghcr.io/paradigmxyz/op-reth:nightly`